src/reducers/entities/teams.ts

```diff
@@ -50,6 +50,16 @@ function teams(state: IDMappedObjects<Team> = {}, action: GenericAction) {
 
         return state;
     }
+    case TeamTypes.RECEIVED_TEAM_UNARCHIVED: {
+        const {teamId} = action.data;
+        const team = state[teamId];
+
+        if (!team) {
+            return state;
+        }
+
+        return {...state, [teamId]: {...team, delete_at: 0}};
```

Comment:
We remove the team from the store when we receive `RECEIVED_TEAM_DELETED` action.
Do we have a way to add it back in the `RECEIVED_TEAM_UNARCHIVED` action?